istio-operator/istio-operator.spec

```diff
@@ -78,10 +94,20 @@ cd tmp/_output/bin/
     cp -pav stripped/istio-operator $RPM_BUILD_ROOT%{_bindir}/
 %endif
 
+# install the charts
+mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/istio-operator/1.1.0
```

Comment:
Can we move the 1.1.0 stuff to a global?

Reply:
Eventually.  It's currently hard-coded in the operator itself.

Reply:
By global, I meant global variable. Set it in one place in the file and we can update it on releases.